### ISSUE TITLE
Switch from ACE to CodeMirror?

### ISSUE THREAD

**Issue by bobrocke** [chuyik/brackets-snippets#49]
Since Brackets is using CodeMirror, is there a way to simplify this snippet extension by using it rather than having to package another JavaScript-based editor?

**Comment by chuyik** [comment]
Yeah you got me. Have to say I was too lazy to do this, but yeah it's better to use CodeMirror instead.
